internal/dispatch/keys/computed_test.go

```diff
@@ -115,7 +117,49 @@ func TestStableCacheKeys(t *testing.T) {
 					},
 				}, computeBothHashes)
 			},
-			"d4a1bda89c9790f50c",
+			"c1db90fecd9c87ed4c",
+		},
+		{
+			"lookup resources with context",
+			func() DispatchCacheKey {
+				return lookupRequestToKey(&v1.DispatchLookupRequest{
+					ObjectRelation: RR("document", "view"),
+					Subject:        ONR("user", "mariah", "..."),
+					Limit:          10,
+					Metadata: &v1.ResolverMeta{
+						AtRevision: "1234",
+					},
+					Context: func() *structpb.Struct {
+						v, _ := structpb.NewStruct(map[string]any{
+							"foo": 1,
+							"bar": true,
+						})
+						return v
+					}(),
+				}, computeBothHashes)
+			},
+			"e3d8ffa3f7aa97e329",
+		},
+		{
+			"lookup resources with different context",
```

Comment:
can we add a test making sure the hash for `nil` context and `zero-value context` are the same? (hopefully 🤞🏻 )

Reply:
If we are really going to move forward with serializing string representation of the context when serialization errors occur, can we at least add testing here?

Reply:
this would be a great spot for adding some fuzzing - please open an issue if you think that makes sense

Reply:
> can we add a test making sure the hash for `nil` context and `zero-value context` are the same? (hopefully 🤞🏻 )

Done

> If we are really going to move forward with serializing string representation of the context when serialization errors occur, can we at least add testing here?

Add testing for what?

> this would be a great spot for adding some fuzzing - please open an issue if you think that makes sense

Not sure we need fuzzing for cache key computation

Reply:
>Add testing for what?

Add a test-case with a Struct that will err on `proto.Marshall` and have the hash key generated

Reply:
Not sure I can force that to error; I suspect that can only really happen if there is a memory issue. Anyway, I changed it to a panic